web/docs/Frontend Starters/Next.js/Next.js + WordPress/setting-environment-variables.md

```diff
@@ -23,3 +23,29 @@ For example:
 WPGRAPHQL_URL=https://my-wordpress-site.pantheon.site/wp/graphql
 IMAGE_DOMAIN=my-image-cdn.site
 ```
+
+## Decoupled Preview
+
+To enable Decoupled Preview, the following environment variables must be set in
+the `.env.development.local` for local dev and in the Pantheon dashboard for
+production or Multidev environments.
+
+```
+PREVIEW_SECRET
+WP_APPLICATION_USERNAME
+WP_APPLICATION_PASSWORD
+PANTHEON_UPLOAD_PATH
+
+```
+
+- `PREVIEW_SECRET` - The secret used on creation of a new preview site in the WP
+  dashboard
+- `WP_APPLICATION_USERNAME` - To be set as the username found and set in the
+  `Users` tab of the WP dashboard
+- `WP_APPLICATION_PASSWORD` - To be set as the password created for the WP
+  application user. Passwords can be set through selecting a user inside the
+  `Users` tab of the WP dashboard
```

Comment:
Use **bold** for navigational elements, see the style guide here: https://pantheon.io/docs/style-guide#bold
```suggestion
- `WP_APPLICATION_USERNAME` - To be set as the username found and set in the
  **Users** tab of the WP dashboard
- `WP_APPLICATION_PASSWORD` - To be set as the password created for the WP
  application user. Passwords can be set through selecting a user inside the
  **Users** tab of the WP dashboard
```